internal/resources/resource_project.go

```diff
@@ -190,6 +200,8 @@ func resourceMetalProjectCreate(d *schema.ResourceData, meta interface{}) (err e
 
 	np := rest.NewProject{
 		Name: d.Get(pName).(string),
+
+		VolumeReplicationEnabled: d.Get(pVolumeReplicationEnabled).(bool),
```

Comment:
need to add tests under https://github.com/HewlettPackard/hpegl-metal-terraform-resources/blob/master/internal/acceptance_test/resource_project_test.go to cover various scenarios.